website/source/api/auth/jwt/index.html.md

```diff
@@ -142,7 +146,15 @@ entities attempting to login. At least one of the bound values must be set.
   "bound_subject": "sl29dlldsfj3uECzsU3Sbmh0F29Fios1@clients",
   "bound_audiences": "https://myco.test",
   "user_claim": "https://vault/user",
-  "groups_claim": "https://vault/groups"
+  "groups_claim": "https://vault/groups",
+  "bound_claims": {
+    "department": "engineering",
+    "sector": "7g"
+  },
+  "claim_mappings" {
```

Comment:
I think this is mad because it's missing a colon.

Reply:
Fixed